emulators/mame/Portfile

```diff
@@ -125,9 +159,15 @@ depends_run-append  \
 # Build Options
 #------------------------------------------------------------------------------
 
-use_parallel_build  yes
+# Note: ARM64 support untested
+#supported_archs     ppc ppc64 i386 x86_64 arm64
+supported_archs     ppc ppc64 i386 x86_64
 
+use_parallel_build  yes
 use_configure       no
+configure.ccache    yes
```

Comment:
don't we want the user's setting for this in macports.conf to decide this for a given installation? You don't see many (any) ports forcing ccache on...

Reply:
Good point, I'll remove this.